meetings/2020-10-13.md

```diff
@@ -4,26 +4,91 @@
 
 ### Attending
 
--
+- Jesse Mapel
+- Stuart Robbins
+- Jay Laura
+- Audrie Fennema
+- Michael Aye
+- Andrew Annex
+- Victor Silva
+- Lauren Adoram Kershner
+- Adam Paquette
 
 ## Agenda / Notes
 
 - Action Items from last meeting
   - Jay will create an issue to collect user stories related to releases and updating
     - Everyone will post their user stories on the issue
+    - create https://github.com/USGS-Astrogeology/ISIS_TC/issues/124
+    - Feedback from 6-10 users
+      - Mixture of stories and people voting existing stories
+    - Not a lot of mission users
+      - Direct outreach
+      - Some additional folks have said they would respond but haven't yet
+    - Want more input
+    - Consider the expertise of different user groups and their ability to work around things
+      - Most feedback is from expert users
+    - How to get more feedback from a braoder group
+      - openplanetary
+      - twitter
+      - email
+      - PEN
+        - repost
+      - PAGs
+      - Facebook
+
   - Jesse and Victor will reach out to a few scientists to contribute to the release working group
+    - Jesse still needs to reach out to ASC scientists
+    - Emerson is willing to participate
 
 - ASC FY21 roadmap
-  - Issue on ISIS_TC repo for collecting questions
+  - Issue on ISIS_TC repo for collecting questions: https://github.com/USGS-Astrogeology/ISIS_TC/issues/125
+  - How does this relate to the TC
+    - Does the TC set priorities for the work? No.
```

Comment:
The "issue" for me isn't so much "setting" the priority, but being able to discuss, comment on, and recommend on priorities for technical work, which I thought is what a technical committee does. I understand that the special triangular situation with the feds/USGS/TC doesn't enable this kind of work easily. 
However, my thinking may be guided by my vision for a Planetary Software Analysis Group (AG) (which I suggested in our white paper as well), where this is the exact procedure: Any NASA AG provides findings through group discussions and these findings are presented to NASA, but may or may not be taken up by NASA.
I just wanted to make it clear that I never demanded "setting" priorities, but rather influencing them by benevolent discussions in the TC.

Reply:
I'll get this moved into the notes. I'll probably paraphrase a bit.